python/test_support/test_folder/test_folder2/hello.txt

```diff
@@ -0,0 +1 @@
+Hello World!
```

Comment:
I wanted to ensure that the recursiveness was working and it seemed a bit heavy handed to distribute the entire `/test_support/sql/` folder using `addFile` - however I'm happy to just use that if you think it's better practice.